Packaging/nix/devilutionx.6

```diff
@@ -0,0 +1,134 @@
+.TH DEVILUTIONX 6 "May 2025" "1.5.4" "DevilutionX Community"
+.SH NAME
+devilutionx \- A source port of Diablo and Hellfire
+.SH SYNOPSIS
+devilutionx [OPTION]
+.SH DESCRIPTION
+.I DevilutionX
+is a source port of
+.B Diablo
+and
+.B Hellfire
+that strives to make it simple to run the game while providing engine improvements, bugfixes, and some optional quality of life features.
+.SH OPTIONS
+.TP
+.B \-h, \-\-help
+Print this message and exit.
+.TP
+.B \-\-version
+Print the version and exit.
+.TP
+.B \-\-data\-dir
+Specify the folder of
+.I diabdat.mpq.
+.TP
+.B \-\-save\-dir
+Specify the folder of save files.
+.TP
+.B \-\-config\-dir
+Specify the location of
+.I diablo.ini.
+.TP
+.B \-\-lang
+Specify the language code (e.g.
+.I en
+or
+.I pt_BR
+).
+.TP
+.B \-n
+Skip startup videos.
+.TP
+.B \-f
+Display frames per second.
+.TP
+.B \-\-verbose
+Enable verbose logging.
+.TP
+.B \-\-record \fI<#>\fR
+Record a demo file.
+.TP
+.B \-\-demo \fI<#>\fR
+Play a demo file.
+.TP
+.B \-\-timedemo
+Disable all frame limiting during demo playback.
+.SH GAME SELECTION
+.TP
+.B \-\-spawn
+Force Shareware mode.
+.TP
+.B \-\-diablo
+Force Diablo mode.
+.TP
+.B \-\-hellfire
+Force Hellfire mode.
+.SH HELLFIRE OPTIONS
+Refer to the official documentation for more details on Hellfire options.
+.SH MULTI-PLAYER
+TCP/IP requires the host to expose port 6112. All games are encrypted and password protected.
+.SH SAVE GAMES AND CONFIGURATIONS
+The configurations and save games are located in:
+.I ~/.local/share/diasurgical/devilution
+.SH INSTALLATION
+To install
+.B DevilutionX:
+.IP 1.
+Extract the files in the archive.
+.IP 2.
+Install
+.B libsdl2.
+.IP 3.
+Copy
+.I DIABDAT.MPQ
+from the CD or GOG-installation (or extract it from the GoG installer) to the DevilutionX folder.
```

Comment:
Should we mention that Battle.net is also a valid option? We don't recommend it as the installation process from there isn't as user frinedly but the MPQ file from there is the same as the other ones.